.github/workflows/build_and_test.yml

```diff
@@ -26,3 +24,27 @@ jobs:
       run: ./gradlew jar
     - name: Test
       run: ./gradlew test
+      
+  build-and-push-anchor-platform:
+    runs-on: ubuntu-latest
+    name: Push stellar/anchor-platform:sha to DockerHub
+    steps:
+    - uses: actions/checkout@v3
+    
+    - name: Get image tag (short sha)
+      shell: bash
+      id: get_tag
+      run: echo ::set-output name=TAG::$(git rev-parse --short ${{ github.sha }} )
```

Comment:
One thing to be aware of is that this image on PRs that gets pushed will have a sha you don't necessarily expect. That's because Github Actions runs workflows on a merge commit with the destination branch merged into the PR branch. This won't affect builds on the main branch, or on the push builds that happen on branches either. If this is going to be a problem let me know and we can talk more about options.